charts/rancher-istio/rancher-istio/100.0.1+up1.10.4/templates/istio-uninstall-job.yaml

```diff
@@ -39,7 +42,9 @@ spec:
             name: istio-installer-overlay
         {{ end }}
       serviceAccountName: istio-installer
+      {{- if .Values.global.rbac.pspEnabled }}
       securityContext:
         runAsUser: 101
         runAsGroup: 101
+      {{- end }}
```

Comment:
1. We are not checking the value of `.Values.global.rbac.pspEnabled` at this place in the `istio-install-job.yaml` file. Should we add it there? 
2. In both files, the `securityContext` at the container lever will be applied anyway, which makes the check useless. I think we should either remove the `securityContext` at the container level or move the checks there.

Reply:
I have moved the security context to the container level for this and am no longer doing the rbac.pspEnabled check as it should always run as user 101

Reply:
Moved back to job context instead of container context.